### PR TITLE
Bump package versions to 1.1.0

### PR DESCRIPTION
**Pull request by prafullbhosale**
Creating a feature/msbuild-1.1.0 branch from feature/msbuild and bumping versions to 1.1.0
Once, this PR is merged, will create a rel/msbuild-1.1.0 branch from this.

cc @natemcmaster @muratg @mlorbetske @pranavkm

### CODE REVIEW

File: src/Microsoft.VisualStudio.Web.CodeGeneration.Tools/Microsoft.VisualStudio.Web.CodeGeneration.Tools.csproj [hunk 1]
```diff
@@ -30,8 +30,9 @@
     <PackageReference Include="Microsoft.Extensions.CommandLineUtils" Version="1.0.0" />
     <PackageReference Include="Microsoft.Extensions.PlatformAbstractions" Version="1.0.0" />
     <PackageReference Include="Microsoft.Extensions.FileProviders.Physical" Version="1.0.0" />
-    <PackageReference Include="Microsoft.DotNet.ProjectModel" Version="1.0.0-rc3-003121" />
+    <!--<PackageReference Include="Microsoft.DotNet.ProjectModel" Version="1.0.0-rc3-003121" />-->
```
Comment (natemcmaster) [inline src/Microsoft.VisualStudio.Web.CodeGeneration.Tools/Microsoft.VisualStudio.Web.CodeGeneration.Tools.csproj]: Why is this commented out?

File: version.props [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <Project>
     <PropertyGroup>
-        <VersionPrefix>1.0.0</VersionPrefix>
-        <VersionSuffix>msbuild4</VersionSuffix>
+        <VersionPrefix>1.1.0</VersionPrefix>
+        <VersionSuffix>msbuild3</VersionSuffix>
```
Comment (natemcmaster) [inline version.props]: ...I assume you mean to get this in for RC.3...does this mean we also need to update dotnet-migrate to know when to use 1.0 vs 1.1?

Comment (prafullbhosale) [inline version.props]: Yes, I believe so.

File: src/Microsoft.VisualStudio.Web.CodeGeneration.Tools/Microsoft.VisualStudio.Web.CodeGeneration.Tools.csproj (hunk 1 above)
Comment (prafullbhosale) [inline src/Microsoft.VisualStudio.Web.CodeGeneration.Tools/Microsoft.VisualStudio.Web.CodeGeneration.Tools.csproj]: Not needed.

**Pull request merged**